dev_guide/builds/build_inputs.adoc

```diff
@@ -173,22 +175,18 @@ endif::[]
 [[source-code]]
 == Git Source
 
-When the `BuildConfig.spec.source.type` is `Git`, a Git repository is
-required.
-ifndef::openshift-online[]
-An inline Dockerfile is optional.
+When specified, source code will be fetched from the location supplied.
 
-The source code is fetched from the location specified and, if the
-`BuildConfig.spec.source.dockerfile` field is specified, the inline Dockerfile
-replaces the one in the `contextDir` of the Git repository.
+ifndef::openshift-online[]
+If an inline Dockerfile is also supplied, the inline Dockerfile
```

Comment:
If an inline Dockerfile is supplied, it will overwrite the Dockerfile (if present) in the `contextDir` of the Git repository.